client/src/pages/CreateUserProfile.js

```diff
@@ -0,0 +1,219 @@
+import { Flex, WhiteSpace } from "antd-mobile";
+import { Dropdown, Menu } from "antd";
+import React from "react";
+import styled, { css } from "styled-components";
+import TextInput from "../components/Input/TextInput";
+import Checkbox from "../components/Input/Checkbox";
+import Label from "../components/Input/Label";
+import UnderlineDescription from "../components/Input/UnderlineDescription";
+import CustomH1 from "../components/Typography/Title/CustomH1";
+import Title from "../components/Typography/Title";
+import SubmitButton from "../components/Button/SubmitButton";
+import { theme, mq } from "../constants/theme";
+
+import PersonalDataImage from "../assets/create-profile-images/personal_data.svg";
+
+require('typeface-dm-sans');
+require('typeface-work-sans');
```

Comment:
For some reason I'm unable to `npm install` this so it's blocking the app from running. So for now in order to see the app I commented this out.

Reply:
what error do you get? Might be problem with package-lock, or the npm module itself?